omrmakefiles/rules.linux.mk

```diff
@@ -187,21 +187,25 @@ ifeq (x86,$(OMR_HOST_ARCH))
         GLOBAL_CXXFLAGS+=-m32 -msse2 -I/usr/include/nptl
         GLOBAL_CPPFLAGS+=-DJ9X86
     endif
+    GLOBAL_CFLAGS+=-fno-strict-aliasing
+    GLOBAL_CXXFLAGS+=-fno-strict-aliasing
 
 else ifeq (aarch64,$(OMR_HOST_ARCH))
-    GLOBAL_CFLAGS+=-march=armv8-a+simd -Wno-unused-but-set-variable
```

Comment:
I think it would be better to set this flag any time we're using gcc
```
ifeq (gcc,$(OMR_TOOLCHAIN))
  GLOBAL_CFLAGS+= -fno-strict-aliasing
  GLOBAL_CXXFLAGS += -fno-strict-aliasting
endif
```